### ISSUE TITLE
Enable/Disable pre & post-evening event functions

### ISSUE THREAD

**Issue by Cao-Jason** [sapmentors/SITregOrganizer#19]
Ability for organizers to either enable the pre-evening event or post-evening event functions, enable both or disable both.

**Comment by skemp353** [comment]
Agree. Not all events have pre/post activities. May be confusing for participants.
